<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminAddNewFood</class>
 <widget class="QDialog" name="adminAddNewFood">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="foodEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="foodPriceLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="cityLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="foodPriceEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="foodLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cityEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="showAll">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
